functions/uninstall.go

```diff
@@ -30,30 +31,38 @@ func Uninstall() ([]error, error) {
 }
 
 // LeaveNetwork - client exits a network
-func LeaveNetwork(network string) ([]error, error) {
+func LeaveNetwork(network string, isDaemon bool) ([]error, error) {
 	faults := []error{}
-	fmt.Println("\nleaving network", network)
 	node, ok := config.Nodes[network]
 	if !ok {
-		fmt.Printf("\nnot connected to network: %s", network)
 		return faults, fmt.Errorf("not connected to network: %s", network)
 	}
-	fmt.Println("deleting node from server")
 	if err := deleteNodeFromServer(&node); err != nil {
 		faults = append(faults, fmt.Errorf("error deleting nodes from server %w", err))
 	}
-	fmt.Println("deleting wireguard interface")
+	// remove node from config
 	if err := deleteLocalNetwork(&node); err != nil {
 		faults = append(faults, fmt.Errorf("error deleting wireguard interface %w", err))
 	}
-	fmt.Println("removing dns entries")
 	if err := removeHostDNS(node.Network); err != nil {
 		faults = append(faults, fmt.Errorf("failed to delete dns entries %w", err))
 	}
-	fmt.Println("restarting daemon")
-	if err := daemon.Restart(); err != nil {
-		faults = append(faults, fmt.Errorf("error restarting daemon %w", err))
+	// re-configure interface if daemon is calling leave
+	if isDaemon {
```

Comment:
doesn't daemon restart accomplish the same thing?

Reply:
well we discussed there is no need to restart proxy/daemon during a node deletion if it can be avoided and that's what the team agreed with